backend/apid/middlewares/refresh_token.go

```diff
@@ -46,42 +47,41 @@ func (m RefreshToken) Then(next http.Handler) http.Handler {
 		payload := &types.Tokens{}
 		err = decoder.Decode(payload)
 		if err != nil {
-			logger.Errorf("could not decode the refresh token: %s", err.Error())
+			logger.WithError(err).Error("could not decode the refresh token")
 			http.Error(w, err.Error(), http.StatusBadRequest)
 			return
 		}
 
 		// Now we want to validate the refresh token
 		refreshToken, err := jwt.ValidateToken(payload.Refresh)
 		if err != nil {
-			logger.Errorf("refresh token is invalid: %s", err.Error())
+			logger.WithError(err).Error("refresh token is invalid")
 			http.Error(w, "Request unauthorized", http.StatusUnauthorized)
 			return
 		}
 
 		// Retrieve the claims for both tokens
 		accessClaims, err := jwt.GetClaims(accessToken)
 		if err != nil {
-			logger.Errorf("could not parse the access token claims: %s", err.Error())
+			logger.WithError(err).Error("could not parse the access token claims")
 			http.Error(w, err.Error(), http.StatusBadRequest)
 			return
 		}
 
 		refreshClaims, err := jwt.GetClaims(refreshToken)
 		if err != nil {
-			logger.Errorf("could not parse the refresh token claims: %s", err.Error())
+			logger.WithError(err).Error("could not parse the refresh token claims")
 			http.Error(w, err.Error(), http.StatusBadRequest)
 			return
 		}
 
 		// Make sure the refresh token belongs to the same user as the access token
 		if accessClaims.Subject == "" || accessClaims.Subject != refreshClaims.Subject {
-			logger.WithField(
-				"user", refreshClaims.Subject,
-			).Errorf("the access and refresh tokens subject do not match: %s != %s",
-				accessClaims.Subject,
-				refreshClaims.Subject,
-			)
+			logger.WithFields(logrus.Fields{
+				"user":            refreshClaims.Subject,
+				"token (access)":  accessClaims.Subject,
```

Comment:
fields are typically searched for, it'd be best if they didn't include characters like spaces or parens. Can we make things like this access_token and refresh_token?